### ISSUE TITLE
Webhooks documentation is incomplete

### ISSUE THREAD

**Issue by rawbee** [wunderlist/api#20]
The documentation for webhooks don't list which actions trigger a webhook call. For instance, my webhook seems to get called in these cases (and there may be more, but these are the ones I've observed):

* a task is created on a list
* a subtask is added to a task on a list
* a note is added to a task on the list

However, there is no call for these actions:

* the task title is modified
* the task is marked as completed
* the tasks on the list are reordered

**Comment by duncan** [comment]
You are absolutely right that we should document this. The other case that I'm aware of no notification for is when a list is deleted. We delete the we hook, but don't send a notification first. 

Thanks for the report. We'll track it and follow up here.

**Comment by passuf** [comment]
@rawbee I agree with the issue about the docs, but are you sure that you do not receive calls for completed tasks? I just tried this by myself and it seems to work.

**Comment by gdiab** [comment]
I don't think I receive a call for completed tasks. Has this not been implemented or am I doing something wrong?

**Comment by rawbee** [comment]
@passuf I am getting calls now. I'm not sure if something was fixed, or if I was doing something wrong, but I am happy to be getting them now :)

**Comment by Sarke** [comment]
Please also document what the `processor_type` and `configuration` parameters do and their possible values.

**Comment by jrgenolsen** [comment]
We still need this :(
How can I use Your web hooks? They call my url, but nothing is inside neither GET or POST. It is just simple url call with no data.

**Comment by Sarke** [comment]
@jrgenolsen The data is a JSON encoded string inside the request body.  Assuming you are using PHP, you can access it like this:

	$json = file_get_contents('php://input');
	$data = json_decode($json, true);

**Comment by khurshid-alam** [comment]
Any news on this....it is still not documented!!

**Comment by gonzie** [comment]
It would be great to get some more documentation on this..

**Comment by hocza** [comment]
Still I could use some documentation on this... :)

**Comment by hocza** [comment]
I could use some documentation on this :(

any news?
